<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">.lst-kix_4vqxw9bvvymd-2>li:before{content:"\0025a0  "}.lst-kix_4vqxw9bvvymd-3>li:before{content:"\0025cf  "}ul.lst-kix_a4yzavjg3805-0{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-5{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-4{list-style-type:none}.lst-kix_4vqxw9bvvymd-7>li:before{content:"\0025cb  "}ul.lst-kix_pvdxjylpjo0w-7{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-6{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-1{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-1{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-0{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-0{list-style-type:none}.lst-kix_4vqxw9bvvymd-0>li:before{content:"\0025cf  "}.lst-kix_4vqxw9bvvymd-1>li:before{content:"\0025cb  "}.lst-kix_4vqxw9bvvymd-8>li:before{content:"\0025a0  "}ul.lst-kix_6qkgmq1i3sis-3{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-3{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-2{list-style-type:none}ul.lst-kix_pvdxjylpjo0w-2{list-style-type:none}.lst-kix_o0d2it6hcwxr-7>li:before{content:"-  "}.lst-kix_6qkgmq1i3sis-2>li:before{content:"\0025a0  "}.lst-kix_o0d2it6hcwxr-8>li:before{content:"-  "}.lst-kix_6qkgmq1i3sis-1>li:before{content:"\0025cb  "}ul.lst-kix_a4yzavjg3805-4{list-style-type:none}ul.lst-kix_a4yzavjg3805-3{list-style-type:none}.lst-kix_4vqxw9bvvymd-6>li:before{content:"\0025cf  "}ul.lst-kix_a4yzavjg3805-2{list-style-type:none}ul.lst-kix_a4yzavjg3805-1{list-style-type:none}.lst-kix_6qkgmq1i3sis-0>li:before{content:"\0025cf  "}ul.lst-kix_a4yzavjg3805-8{list-style-type:none}ul.lst-kix_a4yzavjg3805-7{list-style-type:none}.lst-kix_4vqxw9bvvymd-4>li:before{content:"\0025cb  "}.lst-kix_4vqxw9bvvymd-5>li:before{content:"\0025a0  "}ul.lst-kix_a4yzavjg3805-6{list-style-type:none}ul.lst-kix_a4yzavjg3805-5{list-style-type:none}.lst-kix_a4yzavjg3805-0>li:before{content:"\0025cf  "}.lst-kix_o0d2it6hcwxr-1>li:before{content:"-  "}.lst-kix_a4yzavjg3805-1>li:before{content:"\0025cb  "}.lst-kix_a4yzavjg3805-2>li:before{content:"\0025a0  "}.lst-kix_o0d2it6hcwxr-0>li:before{content:"-  "}.lst-kix_a4yzavjg3805-3>li:before{content:"\0025cf  "}.lst-kix_a4yzavjg3805-4>li:before{content:"\0025cb  "}.lst-kix_o0d2it6hcwxr-6>li:before{content:"-  "}.lst-kix_6qkgmq1i3sis-3>li:before{content:"\0025cf  "}.lst-kix_a4yzavjg3805-7>li:before{content:"\0025cb  "}.lst-kix_a4yzavjg3805-8>li:before{content:"\0025a0  "}.lst-kix_6qkgmq1i3sis-4>li:before{content:"\0025cb  "}.lst-kix_o0d2it6hcwxr-5>li:before{content:"-  "}.lst-kix_6qkgmq1i3sis-5>li:before{content:"\0025a0  "}.lst-kix_a4yzavjg3805-5>li:before{content:"\0025a0  "}.lst-kix_a4yzavjg3805-6>li:before{content:"\0025cf  "}ul.lst-kix_pvdxjylpjo0w-8{list-style-type:none}.lst-kix_o0d2it6hcwxr-4>li:before{content:"-  "}.lst-kix_o0d2it6hcwxr-3>li:before{content:"-  "}.lst-kix_6qkgmq1i3sis-7>li:before{content:"\0025cb  "}.lst-kix_6qkgmq1i3sis-6>li:before{content:"\0025cf  "}.lst-kix_6qkgmq1i3sis-8>li:before{content:"\0025a0  "}.lst-kix_o0d2it6hcwxr-2>li:before{content:"-  "}ul.lst-kix_o0d2it6hcwxr-8{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-7{list-style-type:none}.lst-kix_pvdxjylpjo0w-6>li:before{content:"\0025cf  "}.lst-kix_pvdxjylpjo0w-4>li:before{content:"\0025cb  "}.lst-kix_pvdxjylpjo0w-5>li:before{content:"\0025a0  "}.lst-kix_pvdxjylpjo0w-7>li:before{content:"\0025cb  "}ul.lst-kix_4vqxw9bvvymd-8{list-style-type:none}.lst-kix_pvdxjylpjo0w-8>li:before{content:"\0025a0  "}ul.lst-kix_4vqxw9bvvymd-2{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-3{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-0{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-1{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-6{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-7{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-4{list-style-type:none}ul.lst-kix_4vqxw9bvvymd-5{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-0{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-5{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-4{list-style-type:none}.lst-kix_pvdxjylpjo0w-2>li:before{content:"\0025a0  "}.lst-kix_pvdxjylpjo0w-3>li:before{content:"\0025cf  "}ul.lst-kix_o0d2it6hcwxr-2{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-7{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-1{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-6{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-4{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-3{list-style-type:none}ul.lst-kix_6qkgmq1i3sis-8{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-6{list-style-type:none}ul.lst-kix_o0d2it6hcwxr-5{list-style-type:none}.lst-kix_pvdxjylpjo0w-0>li:before{content:"\0025cf  "}.lst-kix_pvdxjylpjo0w-1>li:before{content:"\0025cb  "}ol{margin:0;padding:0}table td,table th{padding:0}.c0{background-color:#ffffff;color:#333333;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c4{background-color:#ffffff;color:#333333;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:24pt;font-family:"Arial";font-style:normal}.c3{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c9{padding-top:0pt;padding-bottom:8pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c8{padding-top:15pt;padding-bottom:8pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c17{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c20{font-weight:400;text-decoration:none;vertical-align:baseline;font-size:14pt;font-family:"Arial";font-style:normal}.c1{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c15{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c23{font-weight:400;text-decoration:none;vertical-align:baseline;font-size:25.5pt;font-family:"Arial";font-style:normal}.c28{padding-top:8pt;padding-bottom:8pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c22{font-weight:400;text-decoration:none;vertical-align:baseline;font-size:30pt;font-family:"Arial";font-style:normal}.c2{background-color:#ffffff;text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#337ab7;text-decoration:underline}.c10{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c30{font-weight:400;text-decoration:none;vertical-align:baseline;font-family:"Arial"}.c5{background-color:#ffffff;color:#333333;font-weight:700}.c21{max-width:468pt;padding:72pt 72pt 72pt 72pt}.c33{background-color:#333333;color:#ffffff}.c35{margin-left:72pt;padding-left:0pt}.c26{color:inherit;text-decoration:inherit}.c31{font-size:11pt;font-style:normal}.c13{padding:0;margin:0}.c34{color:#000000;font-style:normal}.c16{margin-left:36pt;padding-left:0pt}.c14{border:1px solid black;margin:5px}.c19{color:#4a86e8}.c27{font-style:italic}.c24{background-color:#ffff00}.c12{font-size:24pt}.c11{height:11pt}.c6{color:#333333}.c32{font-size:25.5pt}.c29{font-size:14pt}.c18{margin-left:36pt}.c25{font-size:30pt}.c7{background-color:#ffffff}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c7 c21"><p class="c8"><span class="c6 c7 c25">HDX-Viewer - User manual</span></p><p class="c28"><span class="c30 c27 c6 c29 c7">Toulouse Proteomics Infrastructure</span></p><p class="c28"><span class="c27 c6 c29 c7 c30">Fri March 29 2019</span></p><p class="c9"><span class="c2"><a class="c26" href="https://www.google.com/url?q=https://masstools.ipbs.fr/hdx-viewer&amp;sa=D&amp;ust=1556281108497000">HDX-Viewer</a></span><span class="c6 c7">&nbsp;maps HDX-MS data originating from DynamX or HDExaminer software suites onto 3D structures. It allows the instant inspection of results from individual samples, or the comparison of two experimental conditions. It can be found on our </span><span class="c2"><a class="c26" href="https://www.google.com/url?q=https://masstools.ipbs.fr/mstools/&amp;sa=D&amp;ust=1556281108497000">team&#39;s webpage</a></span><span class="c6 c7">&nbsp;and the corresponding scripts are accessible </span><span class="c2"><a class="c26" href="https://www.google.com/url?q=https://github.com/david-bouyssie/HDX-Viewer&amp;sa=D&amp;ust=1556281108497000">here</a></span><span class="c6 c7">. It is released under the </span><span class="c2"><a class="c26" href="https://www.google.com/url?q=http://www.cecill.info/&amp;sa=D&amp;ust=1556281108498000">CeCILL license</a></span><span class="c0">.</span></p><p class="c8"><span class="c4">Step-by-step description of the options:</span></p><p class="c8"><span class="c4">Input Data:</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 369.00px; height: 69.00px;"><img alt="" src="images/image10.png" style="width: 369.00px; height: 69.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c20 c6 c7">Load Demo Files:</span></p><p class="c9"><span class="c6 c7 c32">&nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 102.00px; height: 33.00px;"><img alt="" src="images/image8.png" style="width: 102.00px; height: 33.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9"><span class="c6 c7">It is possible to click on the &ldquo;</span><span class="c6 c7">Load Demo Files&rdquo;</span><span class="c6 c7">&nbsp;button to test the application and all of the functionalities without loading any input file. The </span><span class="c6 c7">demonstration files correspond</span><span class="c6 c7">&nbsp;to the differential analysis of the SecB tetramer deuterated alone or in the presence of the ChAD peptide </span><span class="c6 c7">(Guillet </span><span class="c6 c7 c27">et al,</span><span class="c6 c7">&nbsp;2019 </span><span class="c27 c6 c7">Nature Commun</span><span class="c6 c7">)</span><span class="c6 c7">. The data represent the subtraction of deuteration uptakes from both data sets, ranging from -1% to 9.5%. A positive difference (in red) corresponds to regions that are more deuterated in the </span><span class="c6 c7">tetramer alone </span><span class="c27 c6 c7">i.e.</span><span class="c6 c7">&nbsp;</span><span class="c6 c7">regions that are protected (less deuterated) upon peptide binding</span><span class="c6 c7">.</span></p><p class="c9"><span class="c6 c7">Upon clicking on the button, </span><span class="c6 c7">three options appear</span><span class="c0">:</span></p><ul class="c13 lst-kix_pvdxjylpjo0w-0 start"><li class="c17 c16"><span class="c6 c7">&ldquo;DynamX&rdquo;:</span><span class="c0">&nbsp;uploads input files coming from the DynamX software suite.</span></li><li class="c17 c16"><span class="c6 c7">&ldquo;HDExaminer&rdquo;:</span><span class="c6 c7">&nbsp;uploads input files coming from the </span><span class="c6 c7">HDExaminer</span><span class="c0">&nbsp;software suite.&rdquo;</span></li><li class="c17 c16"><span class="c6 c7">&ldquo;DOWNLOAD&rdquo;: </span><span class="c0">downloads the demonstration files to the user&rsquo;s computer in order to check the input file format required to run the application.</span></li></ul><p class="c8"><span class="c20 c6 c7">Upload Files:</span></p><p class="c9"><span class="c0">The HDX-MS and .pdb files are uploaded separately.</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 121.00px; height: 33.00px;"><img alt="" src="images/image2.png" style="width: 121.00px; height: 33.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c15 c11"><span class="c3"></span></p><p class="c9"><span class="c6 c7">Upon clicking on the button,</span><span class="c6 c7">&nbsp;two browsers appear</span><sup><a href="#cmnt1" id="cmnt_ref1">[a]</a></sup><sup><a href="#cmnt2" id="cmnt_ref2">[b]</a></sup><span class="c0">:</span></p><ul class="c13 lst-kix_a4yzavjg3805-0 start"><li class="c17 c16"><span class="c6 c7">&ldquo;.pml&rdquo;</span><span class="c0">&nbsp;uploads HDX-MS input files coming from experiments acquired with DynamX or HDExaminer software suites. Both are Python-coded scripts designed to map user-defined colors to the residues of a protein 3D structure:</span></li></ul><ul class="c13 lst-kix_a4yzavjg3805-1 start"><li class="c17 c35"><span class="c6 c7">A DynamX .pml file usually corresponds to a full kinetics of deuteration including all the time points from all the condition(s) exported from a single heatmap.</span><sup><a href="#cmnt3" id="cmnt_ref3">[c]</a></sup><span class="c0">&nbsp;</span></li><li class="c17 c35"><span class="c6 c7">HDExaminer generates a single </span><span class="c6 c7">.pml</span><span class="c0">&nbsp;file per time point. </span></li></ul><p class="c17"><span class="c6 c7">In both cases, multiple files can be uploaded at the same time. It is the responsibility of the user to carefully export the data from DynamX/HDExaminer and make sure that the naming and numbering of the chains correspond to the ones in the .pdb file uploaded alongside the HDX-MS results.</span><sup><a href="#cmnt4" id="cmnt_ref4">[d]</a></sup><sup><a href="#cmnt5" id="cmnt_ref5">[e]</a></sup></p><p class="c1"><span class="c0">Example of the ten first lines of a .pml file generated by DynamX:</span></p><p class="c1"><span class="c0">&hellip;&hellip;&hellip;&hellip;...</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/3, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.030</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/4, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.030</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/5, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/6, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/7, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/8, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/9, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/10, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/11, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">alter /SecB//ABCD/12, properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p><p class="c1"><span class="c0">&hellip;&hellip;&hellip;&hellip;&hellip;</span></p><p class="c1"><span class="c6 c7">This script replaces the</span><span class="c6 c7">&nbsp;B-factor</span><sup><a href="#cmnt6" id="cmnt_ref6">[f]</a></sup><sup><a href="#cmnt7" id="cmnt_ref7">[g]</a></sup><sup><a href="#cmnt8" id="cmnt_ref8">[h]</a></sup><sup><a href="#cmnt9" id="cmnt_ref9">[i]</a></sup><sup><a href="#cmnt10" id="cmnt_ref10">[j]</a></sup><sup><a href="#cmnt11" id="cmnt_ref11">[k]</a></sup><span class="c6 c7">&nbsp;(</span><span class="c6 c7">reflecting the atom mobility) </span><span class="c6 c7">of each covered residue from the .pdb file by its deuteration rate. Each line is organised as follows:</span><sup><a href="#cmnt12" id="cmnt_ref12">[l]</a></sup><sup><a href="#cmnt13" id="cmnt_ref13">[m]</a></sup><sup><a href="#cmnt14" id="cmnt_ref14">[n]</a></sup><sup><a href="#cmnt15" id="cmnt_ref15">[o]</a></sup><sup><a href="#cmnt16" id="cmnt_ref16">[p]</a></sup><sup><a href="#cmnt17" id="cmnt_ref17">[q]</a></sup></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c5">Alter /</span><span class="c19 c7">&lt;Name of the object on the .pdb file&gt;</span><span class="c5">//</span><span class="c19 c7">&lt;chain(s) name(s)&gt;</span><span class="c5">/</span><span class="c19 c7">&lt;residue position in the sequence of the corresponding .pdb file&gt;</span><span class="c5">, properties[&ldquo;</span><span class="c19 c7">&lt;experimental condition&gt; &lt;time point&gt;</span><span class="c5">&rdquo;] = </span><span class="c30 c19 c7 c31">&lt;deuteration rate&gt;</span></p><p class="c1 c11"><span class="c0"></span></p><ul class="c13 lst-kix_o0d2it6hcwxr-0 start"><li class="c1 c16"><span class="c19 c7">Name of the object on the .pdb file: </span><span class="c0">In this case, SecB corresponds to the name of the object in the .pdb file. </span></li><li class="c1 c16"><span class="c19 c7">chain(s) name(s): </span><span class="c0">ABCD corresponds to the 4 chains of the tetrameric SecB in the .pdb file. </span></li><li class="c1 c16"><span class="c19 c7">residue position in the sequence of the corresponding .pdb file: </span><span class="c6 c7">&nbsp;numerotation in the .pdb file, which can be different from the one of the user in the case of N-ter tags or specific isoforms (</span><span class="c6 c7">In this example, the first covered residue is the third one</span><span class="c6 c7">). The residues in the .pdb that do not have any values in the HDX-MS file will be attributed a &ldquo;neutral&rdquo; color that can be defined by the user in HDX-Viewer (field &ldquo;</span><span class="c33">Undetected Region Color&rdquo;</span><span class="c0">).</span></li><li class="c1 c16"><span class="c19 c7">experimental condition and time point: </span><span class="c6 c7">here, </span><span class="c6 c7">&ldquo;SecB_apo - SecB_C4 30 sec</span><sup><a href="#cmnt18" id="cmnt_ref18">[r]</a></sup><span class="c0">&rdquo; corresponds to the experimental condition. In this example, it is the subtraction of the deuteration rates between the protein alone (SecB_apo) and the protein incubated with its ligand (SecB_C4) after 30 sec of deuteration. </span></li><li class="c1 c16"><span class="c19 c7">deuteration rate</span><sup><a href="#cmnt19" id="cmnt_ref19">[s]</a></sup><span class="c19 c7">: </span><span class="c7">deuteration rate or difference of deuteration rate, or any other numeric value that the user would like to see color-coded in the 3D output of HDX-Viewer.</span></li></ul><p class="c1 c11 c18"><span class="c0"></span></p><p class="c1"><span class="c0">The example above only shows the first 10 lines of a typical .pml file. The following lines include the results obtained for all the residues at all the time points of the kinetics.</span></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c6 c7">.pml</span><span class="c6 c7">&nbsp;files generated by HDExaminer look like this:</span></p><p class="c1"><span class="c0">&hellip;&hellip;&hellip;&hellip;...</span></p><p class="c1"><span class="c6 c7"># first, a</span><span class="c6 c7">&nbsp;color scale </span><span class="c6 c7">(RGB) </span><span class="c6 c7">is defined</span><span class="c0">, as chosen by the user in HDExaminer </span></p><p class="c1"><span class="c0">set_color deutColor1x0=[0.176,0.212,0.514]</span></p><p class="c1"><span class="c0">set_color deutColor1x1=[0.187,0.222,0.523]</span></p><p class="c1"><span class="c0">set_color deutColor1x2=[0.198,0.232,0.533]</span></p><p class="c1"><span class="c0">set_color deutColor1x3=[0.209,0.242,0.542]</span></p><p class="c1"><span class="c0">set_color deutColor1x4=[0.220,0.253,0.551]</span></p><p class="c1"><span class="c0">set_color deutColor1x5=[0.231,0.263,0.561]</span></p><p class="c1"><span class="c0">set_color deutColor1x6=[0.242,0.273,0.570]</span></p><p class="c1"><span class="c0">set_color deutColor1x7=[0.253,0.283,0.580]</span></p><p class="c1"><span class="c0">set_color deutColor1x8=[0.264,0.293,0.589]</span></p><p class="c1"><span class="c0">set_color deutColor1x9=[0.275,0.304,0.598]</span></p><p class="c1"><span class="c0">set_color deutColor2x0=[0.286,0.314,0.608]</span></p><p class="c1"><span class="c0">&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;.</span></p><p class="c1"><span class="c0">set_color deutColor9x9=[0.795,0.246,0.206]</span></p><p class="c1"><span class="c0">set_color deutColor10x0=[0.792,0.231,0.200]</span></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c0"># the color used for the undetected regions is defined as &ldquo;neutral&rdquo;</span></p><p class="c1"><span class="c0">set_color neutral=[0.784,0.784,0.784]</span></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c0">#the residues are first hidden in Pymol, shown as cartoon and colored with the neutral color</span></p><p class="c1"><span class="c0">hide everything</span></p><p class="c1"><span class="c0">show cartoon</span></p><p class="c1"><span class="c0">color neutral,all</span></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c0"># then the residues with a deuteration value are colored with the color corresponding to their appropriate deuteration range</span></p><p class="c1"><span class="c0">color deutColor1x4,resi 1-1</span></p><p class="c1"><span class="c0">color deutColor1x4,resi 2-2</span></p><p class="c1"><span class="c0">color deutColor1x4,resi 3-3</span></p><p class="c1"><span class="c0">color deutColor4x4,resi 4-4</span></p><p class="c1"><span class="c0">color deutColor4x4,resi 5-5</span></p><p class="c1"><span class="c0">color deutColor6x2,resi 6-6</span></p><p class="c1"><span class="c0">color deutColor6x2,resi 7-7</span></p><p class="c1"><span class="c0">&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;&hellip;...</span></p><p class="c1"><span class="c0">color deutColor5x9,resi 179-179</span></p><p class="c1"><span class="c0">color deutColor5x9,resi 180-180</span></p><p class="c1"><span class="c0">color deutColor5x9,resi 181-181</span></p><p class="c1 c11"><span class="c0"></span></p><p class="c1"><span class="c0">#finally, the residues are unselected</span></p><p class="c1"><span class="c0">deselect</span></p><p class="c1 c11"><span class="c0"></span></p><ul class="c13 lst-kix_4vqxw9bvvymd-0 start"><li class="c17 c16"><span class="c6 c7">The &ldquo;PDB&rdquo; button</span><span class="c6 c7">&nbsp;uploads the .pdb file provided by the user. Only one .pdb file can be uploaded at once. If you want to plot HDX-MS data on several .pdb files, you need to concatenate them before uploading the resulting .pdb file in HDX-Viewer and visualise the combined structures.</span><sup><a href="#cmnt20" id="cmnt_ref20">[t]</a></sup></li></ul><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 108.00px; height: 34.00px;"><img alt="" src="images/image12.png" style="width: 108.00px; height: 34.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c9"><span class="c0">Upon clicking on the button &ldquo;Download Results&rdquo;, a browser appears to choose where to save the corresponding results as a .zip file.</span></p><p class="c9"><span class="c0">This .zip file contains:</span></p><ul class="c13 lst-kix_6qkgmq1i3sis-0 start"><li class="c9 c16"><span class="c0">the .fasta of the protein(s) extracted from the .pdb file. Be aware that this may not be the entire protein sequence since only the sequence portions present in the .pdb file will be reported. </span></li><li class="c9 c16"><span class="c0">one .pdb file per time point of the kinetics (with the HDX-MS results stored in the B-factor of each amino-acid). This file can be used in other 3D visualisation software for further analysis.</span></li></ul><p class="c1"><span class="c0">Selection of the conditions to visualize:</span></p><p class="c1"><span class="c6 c7">Once the input files have been uploaded, a scrolling menu appears that enables to chose the condition and time point to visualize among all the time points of the </span><span class="c6 c7">kinetics. These are automatically retrieved in the &ldquo;</span><span>properties[&quot;</span><span class="c19 c7">&lt;experimental condition&gt; &lt;time point&gt;</span><span>&quot;]</span><span>&rdquo; field of the DynamX .pml file or from the name of the HDExaminer .pml files (</span><span class="c19 c7">&lt;experimental condition&gt;</span><span>_PyMOL_</span><span class="c7 c19">&lt;time point&gt;</span><span>.pml).</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 395.00px; height: 123.00px;"><img alt="" src="images/image5.png" style="width: 395.00px; height: 123.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c4">Viewer Options:</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 371.00px; height: 434.00px;"><img alt="" src="images/image9.png" style="width: 371.00px; height: 434.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c20 c6 c7">Deuteration Range:</span></p><p class="c1"><span class="c3">Enter the minimum and maximum deuteration values (%) that you want to use as the starting and ending point of the color scale. Click on the Apply button to execute. By default, the minimum and maximum values are determined from the .pml file.</span></p><p class="c8"><span class="c20 c6 c7">Color Scale:</span></p><p class="c1"><span class="c3">Choose in the scrolling menu between the different color scales that are available</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 113.00px; height: 93.00px;"><img alt="" src="images/image3.png" style="width: 113.00px; height: 93.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span>We should distinguish the linear scales (&ldquo;custom&rdquo; / &rdquo;Blue-Red&rdquo; / &rdquo;Blue-White-Red&rdquo; / &rdquo;Red-White-Blue&rdquo;) applied to DynamX results (.pml file) </span><span>from discrete scales </span><span class="c24">(default / XXX / XXX / XXX</span><span>) used for HDExaminer results.</span><sup><a href="#cmnt21" id="cmnt_ref21">[u]</a></sup></p><p class="c1"><span class="c3">&ldquo;Custom&rdquo;: enables you to choose the colors corresponding to the minimum (&ldquo;Min Value Color&rdquo;) and maximum (&ldquo;Max Value Color&rdquo;) values.</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 231.00px; height: 70.00px;"><img alt="" src="images/image6.png" style="width: 231.00px; height: 70.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c3">&ldquo;Blue-Red&rdquo;/&rdquo;Blue-White-Red&rdquo;/&rdquo;Red-White-Blue&rdquo; are three pre-defined linear scales.</span></p><p class="c1"><span class="c3">&ldquo;Default&rdquo;: uses the set of colors chosen by the user in HDExaminer before export.</span></p><p class="c1"><span>&ldquo;</span><span class="c24">XXX&rdquo;/&rdquo;XXX&rdquo;/&rdquo;XXX</span><span class="c3">&rdquo;: three alternative discrete color scales that can be used for HDExaminer data.</span></p><p class="c8"><span class="c6 c29 c7">Undetected</span><span class="c20 c6 c7">&nbsp;Region Color:</span></p><p class="c1"><span>This feature enables to change the color of the residues that have no deuteration value (not detected in the HDX-MS experiment). The user can choose either grey, black or white in the scrolling menu. Importantly, any residue present in the .pdb file but not mentioned in the .pml file will be considered as undetected. </span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 231.00px; height: 29.00px;"><img alt="" src="images/image4.png" style="width: 624.00px; height: 351.00px; margin-left: -24.00px; margin-top: -122.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c20 c6 c7">Background Color:</span></p><p class="c1"><span class="c3">This feature enables to change the color of the background. The user can choose either black or white in the scrolling menu.</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 231.00px; height: 29.00px;"><img alt="" src="images/image4.png" style="width: 624.00px; height: 351.00px; margin-left: -27.00px; margin-top: -150.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c6 c29 c7">Representation</span><sup><a href="#cmnt22" id="cmnt_ref22">[v]</a></sup><span class="c6 c7 c20">:</span></p><p class="c1"><span class="c3">This feature enables to change the representation of the 3D structure. The user can choose among 15 different types of representation including cartoon, surface, rope, sticks...</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 231.00px; height: 29.00px;"><img alt="" src="images/image4.png" style="width: 624.00px; height: 351.00px; margin-left: -23.00px; margin-top: -189.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c20 c6 c7">Spinning options:</span></p><p class="c1"><span class="c3">This feature enables to rotate the 3D structure by pressing the &ldquo;Start Spinning&rdquo; button. The rotation speed can be adjusted in the field &ldquo;Angle (radians)&rdquo; field. 0.005 radians is set as the default value.</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 361.00px; height: 66.00px;"><img alt="" src="images/image4.png" style="width: 624.00px; height: 351.00px; margin-left: -25.00px; margin-top: -228.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c20 c6 c7">Animation options:</span></p><p class="c1"><span class="c3">This feature enables to visualize the different time points of deuteration one after the other, creating a timelapse by clicking on the &ldquo;Start Spinning&rdquo; button and after adjusting the speed in the &ldquo;Delay (ms)&rdquo; field. 1,000 ms is set as the default value. A continuous loop can be obtained by clicking on the &ldquo;Loop forever&rdquo; box. The animation feature can be combined with the spinning feature.</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 366.00px; height: 93.00px;"><img alt="" src="images/image1.png" style="width: 624.00px; height: 351.00px; margin-left: -25.00px; margin-top: -217.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c8"><span class="c12 c6 c7">NGL</span><sup><a href="#cmnt23" id="cmnt_ref23">[w]</a></sup><span class="c12 c6 c7">&nbsp;Viewer and options:</span></p><p class="c15"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 616.50px; height: 536.09px;"><img alt="" src="images/image7.png" style="width: 1349.81px; height: 758.74px; margin-left: -456.43px; margin-top: -146.99px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c15"><span class="c3">The user can zoom in and out, displace and rotate the structure with the mouse control. </span></p><p class="c15"><span class="c3">A full screen visualization can be obtained by clicking on the &ldquo;Full Screen Mode&rdquo; button.</span></p><p class="c15"><span class="c3">At any time, a .png file can be obtained by clicking on the &ldquo;Export As PNG&rdquo; button.</span></p><p class="c15"><span class="c3">Similarly, when using the animation option, spinning option, or both, a .gif file can be obtained by clicking on the &ldquo;Record GIF&rdquo; button. The user can &nbsp;stop the recording by clicking a second time on this button. The corresponding .gif automatically opens in a new window/tab of the internet browser.</span></p><p class="c15"><span class="c3">At any time, an orientation can be saved by clicking on the &ldquo;Save Orientation&rdquo; button. It can be restored later by clicking on the &ldquo;Restore Orientation&rdquo; button.</span></p><p class="c8"><span class="c6 c7 c12">MSA</span><sup><a href="#cmnt24" id="cmnt_ref24">[x]</a></sup><span class="c4">&nbsp;Sequence Viewer and options:</span></p><p class="c1"><span>The protein sequences extracted from the .pdb corresponding to the different chains are visualized with the MSA Viewer at the bottom of the page. The user can scroll along the sequences by clicking and dragging with the mouse.</span></p><p class="c8"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 620.68px; height: 91.17px;"><img alt="" src="images/image11.png" style="width: 1241.47px; height: 698.94px; margin-left: 0.00px; margin-top: -437.42px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c3">The color coding of each residue of the different chains reflects the deuteration rate using the colors parametrized previously (these are the same as the ones in the 3D viewer). In the example presented here, the 4 sequences are colored identically because the .pdb file corresponds to a homotetrameric assembly (4 chains with identical sequences). In the case of an hetero-oligomeric complex, each protein sequence is colored differently, according to its deuteration rates.</span></p><p class="c8"><span class="c20 c6 c7">Sorting:</span></p><p class="c8"><span class="c20 c6 c7">Selection:</span></p><p class="c8"><span class="c20 c6 c7">Vis. elements:</span></p><p class="c8"><span class="c20 c6 c7">Color Scheme:</span></p><p class="c8"><span class="c20 c6 c7">Extra:</span></p><p class="c8"><span class="c20 c6 c7">Export:</span></p><p class="c8"><span class="c6 c7 c29">Help:</span><sup><a href="#cmnt25" id="cmnt_ref25">[y]</a></sup><sup><a href="#cmnt26" id="cmnt_ref26">[z]</a></sup><sup><a href="#cmnt27" id="cmnt_ref27">[aa]</a></sup></p><p class="c8 c11"><span class="c20 c6 c7"></span></p><p class="c8"><span class="c4">Reference:</span></p><p class="c1"><span class="c6">Guillet V, Bordes P, Bon C, Marcoux J, Gervais V, Julie Sala A, Dos Reis S, Slama N, Mares-Mej&iacute;a I, Cirinesi AM, Genevaux P, Mourey L (2019) &ldquo;Structural insights into chaperone addiction of toxin-antitoxin systems&rdquo; Nature Communications 10(1):782</span></p><div class="c14"><p class="c10"><a href="#cmnt_ref1" id="cmnt1">[a]</a><span class="c3">vraiment ? Ce n&#39;est pas encore le cas. Il y aura vraiment 2 browsers qui s&#39;ouvriront en m&ecirc;me temps ?</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref2" id="cmnt2">[b]</a><span class="c3">question technique, je laisse &agrave; David le soin de r&eacute;pondre. Deux boutons (&quot;Upload .pml&quot; et &quot;Upload .pdb&quot;) &ccedil;a me va aussi. faisons au plus simple!</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref3" id="cmnt3">[c]</a><span class="c3">nouveau bug detect&eacute;: il faut que le pml corresponde &agrave; la heatmap d&#39;une seule condition ou bien de la diff&eacute;rence entre 2 conditions, mais pas de plusieurs conditions individuelles (cet export est possible dans DynamX mais il n&#39;est pas pour l&#39;instant compatible avec HDXViewer).</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref4" id="cmnt4">[d]</a><span class="c3">pour l&#39;instant si la num&eacute;rotation des r&eacute;sidus est n&eacute;gative (&ccedil;a arrive souvent quand il y a un tag: on shift le 1er acide amin&eacute; pour commencer sur la Met Nter). Donc soit on fixe le bug (en acceptant des valeurs n&eacute;gatives, soit on dit au user de ne mettre que des valeurs &gt;=0 (0 &ccedil;a marche).</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref5" id="cmnt5">[e]</a><span class="c3">good point!</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref6" id="cmnt6">[f]</a><span class="c3">David , il faudrait peut-&ecirc;tre dire quelque part que notre algo reset tous les B factor &agrave; la valeur de -1 avant de remplacer par les valeurs HDX (ceci permet ensuite de selectionner les &quot;uncovered residues&quot; comme &eacute;tant ceux ayant encore un B-factor &agrave; -1)</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref7" id="cmnt7">[g]</a><span class="c3">Au fait, je vous l&#39;ai jamais dit mais je pense que &quot;covered&quot; et &quot;uncovered&quot; c&#39;est bizarre. @julien, c&#39;est souvent utilis&eacute; dans le domaine ? Si c&#39;est pas le cas je pense qu&#39;il faut changer car c&#39;est pas clair (j&#39;avais pas compris avant que tu ne me l&#39;expliques).</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref8" id="cmnt8">[h]</a><span class="c3">Sinon je suis d&#39;accord avec ton commentaire</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref9" id="cmnt9">[i]</a><span class="c3">on parle vraiment tout le temps de coverage dans le milieu. mais effectivement apr&egrave;s v&eacute;rification on n&#39;utilise pas le terme uncovered mais plut&ocirc;t &quot;regions that are not covered by peptide mapping&quot;....Ca fait un peu long pour un bouton....Pour moi missing &ccedil;a fait tout de suite r&eacute;f&eacute;rence &agrave; une r&eacute;gion qui manque dans la structure/pdb...</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref10" id="cmnt10">[j]</a><span class="c3">Bon du coup le mieux c&#39;est &quot;undetected regions&quot; je pense...</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref11" id="cmnt11">[k]</a><span class="c3">le pb c&#39;est que &quot;uncover&quot; &ccedil;a a un autre sens, du coup &ccedil;a peut porter &agrave; confusion. Va pour &quot;undetected&quot; !</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref12" id="cmnt12">[l]</a><span class="c3">David , il faudrait peut-&ecirc;tre dire quelque part que notre algo reset tous les B factor &agrave; la valeur de -1 avant de remplacer par les valeurs HDX (ceci permet ensuite de selectionner les &quot;uncovered residues&quot; comme &eacute;tant ceux ayant encore un B-factor &agrave; -1)</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref13" id="cmnt13">[m]</a><span class="c3">Au fait, je vous l&#39;ai jamais dit mais je pense que &quot;covered&quot; et &quot;uncovered&quot; c&#39;est bizarre. @julien, c&#39;est souvent utilis&eacute; dans le domaine ? Si c&#39;est pas le cas je pense qu&#39;il faut changer car c&#39;est pas clair (j&#39;avais pas compris avant que tu ne me l&#39;expliques).</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref14" id="cmnt14">[n]</a><span class="c3">Sinon je suis d&#39;accord avec ton commentaire</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref15" id="cmnt15">[o]</a><span class="c3">on parle vraiment tout le temps de coverage dans le milieu. mais effectivement apr&egrave;s v&eacute;rification on n&#39;utilise pas le terme uncovered mais plut&ocirc;t &quot;regions that are not covered by peptide mapping&quot;....Ca fait un peu long pour un bouton....Pour moi missing &ccedil;a fait tout de suite r&eacute;f&eacute;rence &agrave; une r&eacute;gion qui manque dans la structure/pdb...</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref16" id="cmnt16">[p]</a><span class="c3">Bon du coup le mieux c&#39;est &quot;undetected regions&quot; je pense...</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref17" id="cmnt17">[q]</a><span class="c3">le pb c&#39;est que &quot;uncover&quot; &ccedil;a a un autre sens, du coup &ccedil;a peut porter &agrave; confusion. Va pour &quot;undetected&quot; !</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref18" id="cmnt18">[r]</a><span class="c3">@david: il y a un format sp&eacute;cial pour ce Chanel ? Quand j&#39;ai essay&eacute; de faire des exports .pml il fallait que le time point soit en dernier. Si c&#39;est le cas et s&#39;il y a d&#39;autres trucs &agrave; savoir il faudrait l&#39;indiquer.</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref19" id="cmnt19">[s]</a><span class="c3">@David: il y a une limite de nombre de digit. Il faut la mentioner car si on se plante il n&#39;y a pas de message d&#39;erreur compr&eacute;hensif. Et s&#39;il y a d&#39;autres choses restrictives dans le format, merci de les rajouter ;)</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref20" id="cmnt20">[t]</a><span class="c3">&agrave; moins qu&#39;on permettre au user d&#39;upload plusieurs .pdb files, &ccedil;a serait beaucoup plus simple (pour le user...)</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref21" id="cmnt21">[u]</a><span class="c3">en fait, HDExaminer g&eacute;n&egrave;re 92 couleurs (10 de 0 &agrave; 10%, 10 de 10 &agrave; 20% .... + celle du 100% et celle du neutral). Si on coche linear gradient, on utilise les 92 couleurs, si on d&eacute;coche alors on n&#39;a plus que 12 couleurs (1 de 0 &agrave; 10%, 1 de 10 &agrave; 20 % etc.... + 1 pour 100% et 1 pour le neutral). L&agrave; o&ugrave; c&#39;est sioux c&#39;est qu&#39;on peut changer &nbsp;chacune des couleurs mais surtout les ranges (on peut faire de 0 &agrave; 7 % puis de 7 &agrave; 8% etc, donc impredictible). +david.bouyssie@ipbs.fr Je propose donc d&#39;utiliser les couleurs export&eacute;es (et de ne pas chercher &agrave; anticiper ce qui est exporter).</span></p><p class="c10"><span class="c3">_Assigned to David Bouyssi&eacute;_</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref22" id="cmnt22">[v]</a><span class="c3">j&#39;aurais mis &quot;representation type&quot; mais bon, c&#39;est moi..</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref23" id="cmnt23">[w]</a><span class="c3">@David, on peut peut-&ecirc;tre ajouter la ref ?</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref24" id="cmnt24">[x]</a><span class="c3">@David: ref ?</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref25" id="cmnt25">[y]</a><span class="c3">je ne sais pas &agrave; quel point &ccedil;a vaut le coup d&#39;expliquer ces differentes options qui sont intrins&egrave;que au viewer MSA et qui du coup doivent &ecirc;tre (mieux) expliqu&eacute;es ailleurs...</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref26" id="cmnt26">[z]</a><span class="c3">@David: Tu peux peut-&ecirc;tre ajouter un lien vers la description des options du viewer MSA. D&#39;ailleurs, ce serait aussi bien d&#39;ajouter les ref. des outils sur lesquels on se base. Pour le 3D viewer aussi.</span></p></div><div class="c14"><p class="c10"><a href="#cmnt_ref27" id="cmnt27">[aa]</a><span class="c3">Par contre il faudrait aussi ajouter les options quand on clique sur les r&eacute;sidus de la s&eacute;quence et que cela zoome sur le r&eacute;sidu en question sur la structure (David voit de quoi je parle!)</span></p></div></body></html>