
<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<meta charset="UTF-8">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;}
h1
	{margin-top:20.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:20.0pt;
	font-family:"Arial",sans-serif;
	font-weight:normal;}
h2
	{margin-top:18.0pt;
	margin-right:0cm;
	margin-bottom:6.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Arial",sans-serif;
	font-weight:normal;}
h3
	{margin-top:16.0pt;
	margin-right:0cm;
	margin-bottom:4.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Arial",sans-serif;
	color:#434343;
	font-weight:normal;}
h4
	{margin-top:14.0pt;
	margin-right:0cm;
	margin-bottom:4.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:12.0pt;
	font-family:"Arial",sans-serif;
	color:#666666;
	font-weight:normal;}
h5
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:4.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;
	color:#666666;
	font-weight:normal;}
h6
	{margin-top:12.0pt;
	margin-right:0cm;
	margin-bottom:4.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;
	color:#666666;
	font-weight:normal;
	font-style:italic;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{mso-style-link:"Commentaire Car";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoTitle, li.MsoTitle, div.MsoTitle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:3.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:26.0pt;
	font-family:"Arial",sans-serif;}
p.MsoSubtitle, li.MsoSubtitle, div.MsoSubtitle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:16.0pt;
	margin-left:0cm;
	line-height:115%;
	page-break-after:avoid;
	font-size:15.0pt;
	font-family:"Arial",sans-serif;
	color:#666666;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p
	{margin-right:0cm;
	margin-left:0cm;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{mso-style-link:"Objet du commentaire Car";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-link:"Texte de bulles Car";
	margin:0cm;
	margin-bottom:.0001pt;
	font-size:9.0pt;
	font-family:"Segoe UI",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0cm;
	margin-right:0cm;
	margin-bottom:0cm;
	margin-left:36.0pt;
	margin-bottom:.0001pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;}
span.TextedebullesCar
	{mso-style-name:"Texte de bulles Car";
	mso-style-link:"Texte de bulles";
	font-family:"Segoe UI",sans-serif;}
span.CommentaireCar
	{mso-style-name:"Commentaire Car";
	mso-style-link:Commentaire;}
span.ObjetducommentaireCar
	{mso-style-name:"Objet du commentaire Car";
	mso-style-link:"Objet du commentaire";
	font-weight:bold;}
span.apple-tab-span
	{mso-style-name:apple-tab-span;}
.MsoChpDefault
	{font-family:"Arial",sans-serif;}
.MsoPapDefault
	{line-height:115%;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0cm;}
ul
	{margin-bottom:0cm;}
-->
</style>

</head>

<body lang=FR link=blue vlink=purple>

<div class=WordSection1>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:30.0pt;
line-height:115%;color:#333333;background:white'>HDX-Viewer 1.2 - User manual</span></p>

<p class=MsoNormal style='margin-top:8.0pt;margin-right:0cm;margin-bottom:8.0pt;
margin-left:0cm;text-align:justify'><i><span lang=EN-US style='font-size:14.0pt;
line-height:115%;color:#333333;background:white'>Toulouse Proteomics
Infrastructure</span></i></p>

<p class=MsoNormal style='margin-top:8.0pt;margin-right:0cm;margin-bottom:8.0pt;
margin-left:0cm;text-align:justify'><i><span lang=EN-US style='font-size:10.0pt;
line-height:115%;color:#333333;background:white'>Contact:
https://github.com/david-bouyssie/hdx-viewer/issues</span></i></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify'><span style='color:#333333;background:
white'>If you use HDX-Viewer for your research please cite:<br/> 
David Bouyssié*, Jean Lesne*, Marie Locard-Paulet, Renaud Albigot, Odile Burlet-Schiltz, Julien Marcoux: HDX-Viewer: interactive 3D visualization of hydrogen-deuterium exchange data.<br/>
Bioinformatics, btz550, <a href="https://doi.org/10.1093/bioinformatics/btz550" target="_blank" >https://doi.org/10.1093/bioinformatics/btz550</a> <br/><br/> 
*These authors contributed equally
</span></p>


<p class=MsoNormal style='margin-top:8.0pt;margin-right:0cm;margin-bottom:8.0pt;
margin-left:0cm;text-align:justify'><i><span lang=EN style='font-size:14.0pt;
line-height:115%;color:#333333;background:white'>Thursday 4<sup>th</sup> of July
2019</span></i></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify'><span
lang=EN><a href="https://masstools.ipbs.fr/hdx-viewer"><span style='color:#337AB7;
background:white'>HDX-Viewer</span></a><span style='color:#333333;background:
white'> maps onto 3D structures HDX-MS data originating from DynamX or
HDExaminer software suites. Alternatively, it is possible to upload custom-made
CSVs following a predefined template to visualise deuterium uptake values from
any other source. It allows the prompt inspection of results from individual
samples, or the comparison of two experimental conditions. It can be found on
our </span><a href="https://masstools.ipbs.fr/mstools/"><span style='color:
#337AB7;background:white'>team's webpage</span></a><span style='color:#333333;
background:white'> and the corresponding scripts are accessible </span><a
href="https://github.com/david-bouyssie/HDX-Viewer"><span style='color:#337AB7;
background:white'>here</span></a><span style='color:#333333;background:white'>.
It is released under the </span><a href="http://www.cecill.info/"><span
style='color:#337AB7;background:white'>CeCILL</span></a><span style='color:
#333333;background:white'> and </span><a
href="https://www.gnu.org/licenses/lgpl-3.0.en.html"><span style='color:#337AB7;
background:white'>GNU LGPL</span></a><span style='color:#333333;background:
white'> licenses.</span></span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;color:#333333;background:white'>Step-by-step description of
the options:</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;color:#333333;background:white'>Input Data:</span></p>

<p class=MsoNormal><img border=0 width=624 height=190 id="Image 18"
src="./images/image001.jpg"></p>

<p style='margin-top:15.0pt;margin-right:0cm;margin-bottom:8.0pt;margin-left:
0cm;text-align:justify'><span lang=EN-US style='font-size:14.0pt;font-family:
"Arial",sans-serif;color:#333333;background:white'>Input format:</span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;
text-align:justify'><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:#333333;background:white'>HDX-Viewer supports inputs coming from two
different software suites (DynamX and HDExaminer). A predefined template (based
on the HDExaminer CSV output) can also be loaded. The first user action
consists in selecting one of the two radio button options:</span></p>

<p style='margin-top:12.0pt;margin-right:0cm;margin-bottom:0cm;margin-left:
36.0pt;margin-bottom:.0001pt;text-align:justify;text-indent:-18.0pt;background:
white;vertical-align:baseline'><span lang=EN-US style='font-size:10.0pt;
font-family:Symbol;color:#333333'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:#333333;background:white'>“DynamX”: to upload PML files coming from
DynamX.</span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:12.0pt;margin-left:
36.0pt;text-align:justify;text-indent:-18.0pt;background:white;vertical-align:
baseline'><span lang=EN-US style='font-size:10.0pt;font-family:Symbol;
color:#333333'>·<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:#333333;background:white'>“HDExaminer/CSV”: to upload CSV files coming
from HDExaminer or a predefined template</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;color:#333333;background:white'>Load Demo Files:</span></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify'><span
lang=EN style='font-size:25.5pt;line-height:115%;color:#333333;background:white'> </span><span
style='font-size:25.5pt;line-height:115%;color:#333333'><img border=0
width=149 height=35 id=image16.png
src="./images/image002.gif"></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;
text-align:justify'><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:black;background:white'>It is possible to click on the “Load Demo Files”
button to test the application and all of the functionalities without loading
any input file. The demonstration files correspond to the differential analysis
of the SecB tetramer deuterated alone or in the presence of the ChAD peptide
(Guillet <i>et al,</i> 2019 <i>Nature Commun</i>). The data represent the
subtraction of deuteration uptakes from both data sets, ranging from -1% to
9.5%. A positive difference (in red) corresponds to regions that are more
deuterated in the tetramer alone, <i>i.e.</i> regions that are protected (less
deuterated) upon peptide binding. The demonstration files can be loaded either
in the DynamX format or in the HDExaminer format (depending on the radio button
that is selected). Please be aware we did not perform the same manual curation
on the two provided files, so the computed deuteration uptakes are not
identical, and the HDX-Viewer visualizations of DynamX and HDExaminer
demonstration files may differ.</span></p>

<p style='margin-top:15.0pt;margin-right:0cm;margin-bottom:8.0pt;margin-left:
0cm;text-align:justify'><span lang=EN-US style='font-size:14.0pt;font-family:
"Arial",sans-serif;color:black;background:white'>Upload Files:</span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;
text-align:justify'><b><i><span lang=EN-US style='font-size:11.0pt;font-family:
"Arial",sans-serif;color:#A61C00;background:white'>Important note:</span></i></b><span
lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;color:black;
background:white'> <b>uploaded files are temporarily stored on our servers and
deleted monthly. If you close your browser, you won’t be able to restore your
previous session and you will have to upload your files again.&nbsp;</b></span></p>

<p style='margin:0cm;margin-bottom:.0001pt;text-align:justify'><span
lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;color:black;
background:white'>PML/CSV and PDB files (HDX-MS and structural data,
respectively) are uploaded separately. Once the adequate PML/CSV format has
been selected (DynamX, HDExaminer/CSV, see “input format” section above), the
second action consists in uploading your PDB file by pressing the “Upload PDB
File” button. Only one PDB file can be uploaded at once. If you want to plot
HDX-MS data on several PDB files, you need to concatenate them before uploading
the resulting PDB file in HDX-Viewer and visualize the combined structures.</span></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN-US style='font-size:
25.5pt;line-height:115%;background:white'> </span><span style='font-size:25.5pt;
line-height:115%'><img border=0 width=186 height=45 id=image7.png
src="./images/image003.gif"></span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;
text-align:justify'><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:black;background:white'>The PDB file upload will initialize a new session
on the HDX-Viewer server. The PDB file will be parsed and a FASTA file
containing the different amino acid chains of the PDB file will be generated.
This FASTA file will be further used by HDX-Viewer to display an overlay of the
amino acid sequence(s) it contains and of the provided deuteration uptakes.</span></p>

<p style='margin-top:0cm;margin-right:0cm;margin-bottom:8.0pt;margin-left:0cm;
text-align:justify'><span lang=EN-US style='font-size:11.0pt;font-family:"Arial",sans-serif;
color:black;background:white'>If the PDB file has been parsed without any error
a new button named “Upload PML/CSV Files” appears.</span></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify'><img
border=0 width=179 height=46 id=image1.png
src="./images/image004.gif"><span
lang=EN-US> or  </span><img border=0 width=188 height=46 id="Image 1"
src="./images/image005.jpg"></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify;line-height:
normal'><span lang=EN-US style='color:black;background:white'>This will allow
you to upload HDX-MS output files coming from experiments acquired with DynamX
(PML) or HDExaminer (CSV) software suites, or a custom CSV based on the
template presented below:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:black;margin-top:12.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><span
     lang=EN-US style='background:white'>The DynamX PML file is a Python-coded
     script designed to map user-defined colors to the residues of a protein 3D
     structure. It corresponds to the full kinetics of deuteration including
     all the time points from the exported heatmap.&nbsp;</span></li>
</ul>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black;background:white'>Example of the ten first lines
of a PML file generated by DynamX:</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='line-height:normal'><span style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/3,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.030</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/4,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.030</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/5,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/6,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/7, properties[&quot;SecB_apo
- SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/8,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/9,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/10,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/11,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>alter /SecB//ABCD/12,
properties[&quot;SecB_apo - SecB_C4 30 sec&quot;] = 0.046</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black;background:white'>This script replaces the
B-factor (reflecting the atom mobility) of each covered residue from the PDB
file by its deuteration rate. Each line is organised as follows:</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><b><span
lang=EN-US style='color:#333333;background:white'>alter /</span></b><span
lang=EN-US style='color:#4A86E8;background:white'>&lt;name of the object on the
PDB file&gt;</span><b><span lang=EN-US style='color:#333333;background:white'>//</span></b><span
lang=EN-US style='color:#4A86E8;background:white'>&lt;chain(s) name(s)&gt;</span><b><span
lang=EN-US style='color:#333333;background:white'>/</span></b><span lang=EN-US
style='color:#4A86E8;background:white'>&lt;residue position in the sequence of
the corresponding PDB file&gt;</span><b><span lang=EN-US style='color:#333333;
background:white'>, properties[“</span></b><span lang=EN-US style='color:#4A86E8;
background:white'>&lt;experimental condition&gt; &lt;time point&gt;</span><b><span
lang=EN-US style='color:#333333;background:white'>”] = </span></b><span
lang=EN-US style='color:#4A86E8;background:white'>&lt;deuteration rate&gt;</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:#333333;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='color:#4A86E8;background:white'>name of the object on the PDB file:
     </span><span lang=EN-US style='background:white'>In this case, SecB
     corresponds to the name of the object in the PDB file.&nbsp;</span></li>
 <li class=MsoNormal style='color:black;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='color:#4A86E8;background:white'>chain(s) name(s): </span><span
     lang=EN-US style='color:#333333;background:white'>ABCD corresponds to the
     4 chains of the tetrameric SecB in the PDB file.&nbsp;</span></li>
 <li class=MsoNormal style='color:#333333;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='color:#4A86E8;background:white'>residue position in the sequence of
     the corresponding PDB file:</span><span lang=EN-US style='background:white'>
     </span><span lang=EN-US style='color:black;background:white'>numerotation
     in the PDB file, which can be different from the one of the user in the
     case of N-ter tags or specific isoforms (In this example, the first
     covered residue is the third one). The residues in the PDB that do not
     have any values in the PML file will be attributed a “neutral” color that
     can be defined by the user in HDX-Viewer (see “</span><span lang=EN-US
     style='color:black'>Undetected Region Color” section below).</span></li>
 <li class=MsoNormal style='color:#333333;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='color:#4A86E8;background:white'>experimental condition and time
     point: </span><span lang=EN-US style='color:black;background:white'>here,
     “SecB_apo - SecB_C4 30 sec” corresponds to the experimental condition. In
     this example, it is the subtraction of the deuteration rates between the
     protein alone (SecB_apo) and the protein incubated with its ligand
     (SecB_C4) after 30 sec of deuteration.&nbsp;</span></li>
 <li class=MsoNormal style='color:#333333;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='color:#4A86E8;background:white'>deuteration rate: </span><span
     lang=EN-US style='color:black;background:white'>deuteration rate or
     difference of deuteration rates, or any other numeric value that the user
     would like to see color-coded in the 3D output of HDX-Viewer.</span></li>
</ul>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black;background:white'>The example above only shows
the first 10 lines of a typical PML file. The following lines include the results
obtained for all the residues at all the time points of the kinetics.</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:black;margin-top:12.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><span
     lang=EN-US style='background:white'>The CSV files generated by HDExaminer
     correspond to the full kinetics of deuteration including all the time
     points from the exported heatmap.&nbsp;</span></li>
</ul>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>The first column contains the residues for which
deuteration values have been acquired. The undetected residues appear with
values set to 0 for each timepoint (see residue 7 below). The second column is
not necessary and has been suppressed in the custom CSV template (see below).
The following columns require the timepoint expressed in sec (and written “Xs”)
in the header as well as the percentage of deuteration for each corresponding
residue. The additional columns with headers entitled “Xs - spread” are not
used by HDX-Viewer.</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black;background:white'>HDExaminer </span><span style='color:black'>CSV
file example:</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>Start,End,5s,10s,5s - spread,10s - spread</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>3,3,0.716,0.676,0.059,0.051</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>4,4,0.741,0.802,0.050,0.047</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>5,5,0.741,0.802,0.050,0.047</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>6,6,0.146,0.164,0.029,0.021</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>7,7,0,0,0.029,0.021</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>8,8,1,1,0.043,0.049</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>9,9,1,1,0.046,0.058</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>10,10,0.393,0.490,0.052,0.063</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>11,11,0.071,0.076,0.043,0.049</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>This should be displayed in your spreadsheet
editor as:</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>Start    End      5s        10s      5s - spread      10s
- spread</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>3          3          0.716   0.676   0.059               0.051</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>4          4          0.741   0.802   0.050               0.047</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>5          5          0.741   0.802   0.050               0.047</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>6          6          0.146   0.164   0.029               0.021</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>7          7          0          0          0.029               0.021</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>8          8          1          1          0.043               0.049</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>9          9          1          1          0.046               0.058</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>10        10        0.393   0.490   0.052               0.063</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>11        11        0.071   0.076   0.043               0.049</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:black;margin-top:12.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><span
     lang=EN-US style='background:white'>The CSV file that can be used as a
     template to manually upload HDX-MS data is a simplified version of
     HDExaminer CSV outputs:&nbsp;</span></li>
</ul>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>Example of the ten first lines of a custom CSV
input:</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>Residues,5s,10s</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>3,0.716,0.676</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>4,0.741,0.802</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>5,0.741,0.802</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>6,0.146,0.164</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>7,0,0</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>8,1,1</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>9,1,1</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>10,0.393,0.490</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>11,0.071,0.076</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black'>This should be displayed in your spreadsheet
editor as:</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>Residues         5s        10s</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>3                      0.716   0.676</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>4                      0.741   0.802</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>5                      0.741   0.802</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>6                      0.146   0.164</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>7                      0          0</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>8                      1          1</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>9                      1          1</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>10                    0.393   0.490</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
style='color:black'>11                    0.071   0.076</span></p>

<div class=MsoNormal align=center style='text-align:center;line-height:normal'><span
style='font-size:12.0pt'>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify;line-height:normal'><span lang=EN-US
style='color:black;background:white'>These PML/CSV files may have been
generated either in differential mode (difference between two experiments) or
non-differential mode (single condition). This will not impact HDX-Viewer
visualization, except that the deuteration range will change in differential
mode since negative values are also expected.</span></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify;line-height:normal'><span lang=EN-US
style='color:black;background:white'>In any case, multiple PML/CSV files can be
uploaded at the same time, or sequentially by clicking multiple times on the
“Upload PML Files” button. Note that each batch of PML/CSV files uploaded to
HDX-Viewer will be merged into a master PML file. The final visualization thus
corresponds to the concatenation of all the uploaded PML/CSV files in the
current HDX-Viewer session.</span></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify;line-height:normal'><b><i><span
style='color:#A61C00;background:white'>Important notes:&nbsp;</span></i></b></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:black;margin-top:12.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><b><span
     lang=EN-US style='background:white'>It is the responsibility of the user
     to carefully export the data from DynamX and make sure that the naming and
     numbering of the chains correspond to the ones in the PDB file uploaded
     together with the PML file(s).&nbsp;</span></b></li>
 <li class=MsoNormal style='color:black;margin-bottom:12.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><b><span
     lang=EN-US style='background:white'>In the case of HDExaminer or the
     predefined template, the different chains of the PDB file are
     automatically detected. A scrolling menu appears and the user must select
     the appropriate chain(s) before uploading the corresponding CSV files.</span></b></li>
</ul>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify'><img border=0 width=518 height=125
id="Image 20" src="./images/image006.jpg"></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN style='background:
white'>&nbsp;</span></p>

<div class=MsoNormal align=center style='text-align:center'><span lang=EN>

<hr size=2 width="100%" align=center>

</span></div>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Download results:</span></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify'><img border=0 width=176 height=43
id=image10.png
src="./images/image007.gif"></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify;line-height:normal'><span lang=EN-US
style='color:black;background:white'>This button can be used to download either
the uploaded files or the demonstration files.</span></p>

<p class=MsoNormal style='margin-top:12.0pt;margin-right:0cm;margin-bottom:
12.0pt;margin-left:0cm;text-align:justify;line-height:normal'><span lang=EN-US
style='color:black;background:white'>Downloading the demonstration files can be
useful to check the input file format required to run the application for the
two supported software.</span></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify;line-height:
normal'><span lang=EN-US style='color:black;background:white'>Upon clicking on
the button “Download Results”, a browser appears to choose where to save the
corresponding results as a ZIP file.</span></p>

<p class=MsoNormal style='margin-bottom:8.0pt;text-align:justify;line-height:
normal'><span style='color:black;background:white'>This ZIP file contains:</span></p>

<ul style='margin-top:0cm' type=disc>
 <li class=MsoNormal style='color:black;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='background:white'>the PML/CSV and PDB used as input files.</span></li>
 <li class=MsoNormal style='color:black;text-align:justify;line-height:normal;
     background:white;vertical-align:baseline'><span lang=EN-US
     style='background:white'>the FASTA sequence(s) of the protein(s) extracted
     from the PDB file. Be aware that this may not correspond to the entire
     protein sequence(s) since only the residues present in the PDB file will
     be reported.&nbsp;</span></li>
 <li class=MsoNormal style='color:black;margin-bottom:8.0pt;text-align:justify;
     line-height:normal;background:white;vertical-align:baseline'><span
     lang=EN-US style='background:white'>one PDB file per time point of the
     kinetics (with the HDX-MS results stored in the B-factor of each
     amino-acid). These PDB files can be used in any other 3D visualization
     software for further analysis.</span></li>
</ul>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='font-size:14.0pt;color:black;background:white'>Selection of
the condition to visualize:</span></p>

<p class=MsoNormal style='line-height:normal'><span lang=EN-US
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='text-align:justify;line-height:normal'><span
lang=EN-US style='color:black;background:white'>Once the input files have been
uploaded, a scrolling menu appears and enables to choose the condition and time
point to visualize among all the conditions/time points of the experiment.</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><img border=0 width=572 height=260
id="Image 21" src="./images/image008.jpg"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;color:#333333;background:white'>Viewer Options:</span></p>

<p class=MsoNormal><img border=0 width=411 height=477 id=image15.png
src="./images/image009.gif"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Deuteration Range:</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='color:black'>Enter
the minimum and maximum deuteration values (%) that you want to use as the
starting and ending points of the color scale. Click on the Apply button to
execute. By default, the minimum and maximum values are determined from the
PML/CSV file.</span><span lang=EN style='font-size:14.0pt;line-height:115%;
background:white'> </span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Color Scale:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Choose between the
different color scales that are available in the scrolling menu: “Blue-Red”,
”Blue-White-Red and ”Red-White-Blue”.</span></p>

<p class=MsoNormal><img border=0 width=223 height=88 id="Image 22"
src="./images/image010.jpg"></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Alternatively, you
can customize the color gradient by choosing “Custom” in the scrolling menu,
and the colors corresponding to its minimum (“Min Value Color”) and maximum
(“Max Value Color”) values.</span></p>

<p class=MsoNormal><img border=0 width=223 height=65 id="Image 23"
src="./images/image011.jpg"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Undetected Region Color:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>This feature
enables to change the color of the residues that have no deuteration value (not
detected in the HDX-MS experiment). The user can choose either grey, black or
white in the scrolling menu. Importantly, any residue present in the PDB file
but not mentioned in the PML/CSV file will be considered as undetected. </span></p>

<p class=MsoNormal style='text-align:justify'><span style='color:#333333'><img
border=0 width=356 height=41 id=image8.png
src="./images/image012.gif"></span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Background Color:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>This feature
enables to change the color of the background. The user can choose either black
or white in the scrolling menu.</span></p>

<p class=MsoNormal><img border=0 width=330 height=38 id=image2.png
src="./images/image013.gif"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Representation:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>This feature
enables to change the representation of the 3D structure. The user can choose
among 15 different types of representation including cartoon, surface, rope,
sticks...</span></p>

<p class=MsoNormal><img border=0 width=330 height=34 id=image11.png
src="./images/image014.gif"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Spinning options:</span></p>

<p class=MsoNormal><span lang=EN style='color:black'>This feature enables to
rotate the 3D structure by pressing the “Start Spinning” button. The rotation
speed can be adjusted in the field “Angle (radians)” (0.005 is set as the
default value).</span><span lang=EN> </span></p>

<p class=MsoNormal><img border=0 width=536 height=94 id=image12.png
src="./images/image015.gif"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:14.0pt;
line-height:115%;background:white'>Animation options:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>This feature
enables to visualize the different time points of deuteration one after the
other, creating a timelapse by clicking on the “Start Spinning” button and
after adjusting the speed in the “Delay (ms)” field. 1,000 ms is set as the
default value. A continuous loop can be obtained by clicking on the “Loop
forever” box. The animation feature can be combined with the spinning feature.</span></p>

<p class=MsoNormal><img border=0 width=540 height=134 id=image17.png
src="./images/image016.gif"></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;background:white'>NGL Viewer:</span></p>

<p class=MsoNormal><img border=0 width=616 height=536 id=image9.png
src="./images/image017.gif"></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>The user can zoom
in and out, move and rotate the structure with the mouse control. </span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>A full screen
visualization can be obtained by clicking on the “Full Screen Mode” button.</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>At any time, a PNG
file can be obtained by clicking on the “Export As PNG” button.</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Similarly, when
using the animation option, spinning option, or both, a GIF file can be
obtained by clicking on the “Record GIF” button. The user can stop the
recording by clicking a second time on this button. The corresponding GIF
automatically opens in a new window/tab of the internet browser.</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>At any time, an
orientation can be saved by clicking on the “Save Orientation” button. It can
be restored later by clicking on the “Restore Orientation” button.</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Double-clicking on
any residue will center the viewer on this residue.</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;background:white'>MSA Sequence Viewer:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>The protein
sequences extracted from the PDB corresponding to the different chains are
visualized with the MSA Viewer at the bottom of the page. The user can scroll
along the sequences by clicking and dragging with the mouse.</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span style='font-size:25.5pt;
line-height:115%'><img border=0 width=624 height=85 id=image4.png
src="./images/image018.gif"></span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Each residue of the
different chains is color-coded with the deuteration rate using the colors
defined for the 3D visualization. In the example presented here, the 4
sequences are colored identically because the PDB file corresponds to a
homotetrameric assembly (4 chains with identical sequences). In the case of an
hetero-oligomeric complex, each protein sequence is colored differently,
according to its deuteration rates. Double-clicking on any residue will
highlight it in green, both on the MSA sequence viewer and NGL viewer. The
different options of the MSA sequence viewer (Sorting / Selection / Vis.
elements / Color scheme / Extras / Export / Help) are described elsewhere (<a
href="https://github.com/wilzbach/msa/wiki/User-manual"><span style='color:
#1155CC'>https://github.com/wilzbach/msa/wiki/User-manual</span></a>).</span></p>

<p class=MsoNormal style='margin-top:15.0pt;margin-right:0cm;margin-bottom:
8.0pt;margin-left:0cm;text-align:justify'><span lang=EN style='font-size:24.0pt;
line-height:115%;background:white'>Reference:</span></p>

<p class=MsoNormal style='text-align:justify'><span lang=EN>Guillet V, Bordes
P, Bon C, Marcoux J, Gervais V, Julie Sala A, Dos Reis S, Slama N, Mares-Mejía
I, Cirinesi AM, Genevaux P, Mourey L (2019) “Structural insights into chaperone
addiction of toxin-antitoxin systems” Nature Communications 10(1):782.</span></p>

</div>

</body>

</html>
